src/components/Home.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React, { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import { FaArrowAltCircleRight } from 'react-icons/fa';
+import { fetchCrypto } from '../redux/crypto/crypto';
+import Loading from './Loading';
+
+const Home = () => {
+  const [search, setSearch] = useState('');
+  const coins = useSelector((state) => state.cryptoData);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchCrypto());
+  }, [dispatch]);
+  const handleSearch = (e) => {
+    setSearch(e.target.value);
+  };
+
+  const filteredCoins = coins.crypto.filter((coin) => (
+    coin.name.toLowerCase().includes(search.toLowerCase())
+  ));
+  if (coins.isLoding || coins.isEror) {
+    return <Loading />;
+  }
+  //   if (coins.isEror) {
+  //     return <h1>Something went wrong...</h1>;
+  //   }
```

Comment:
- [ ] It will be nice to remove commented code from your code base as it makes your code base dirty and less readable.